channeldb/witness_cache.go

```diff
@@ -78,10 +77,10 @@ type witnessEntry struct {
 	witness []byte
 }
 
-// AddSHA256Witnesses adds a batch of new SHA256 preimages into the witness
+// AddSha256Witnesses adds a batch of new sha256 preimages into the witness
 // cache. This is an alias for AddWitnesses that uses Sha256HashWitness as the
 // preimages' witness type.
-func (w *WitnessCache) AddSHA256Witnesses(preimages ...lntypes.Preimage) error {
+func (w *WitnessCache) AddSha256Witnesses(preimages ...lntypes.Preimage) error {
```

Comment:
👍 I was a bit confused by the introduction of the two methods when reading this commit by commit, so would be nice to squash this with the first change :)

Reply:
Ok, I see now that this is planned 👍